### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PranabNandy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,49 @@
...
+
+## Instruction list 
+1. [CGI using C++ on the BeagleBone](http://derekmolloy.ie/beaglebone-cgicc/)
+2. [Introduction to the BeagleBone Black Device Tree](https://learn.adafruit.com/introduction-to-the-beaglebone-black-device-tree/device-tree-overlays)\
+4. [The Beaglebone Black and Device Tree Overlays](http://www.ofitselfso.com/BeagleNotes/Beaglebone_Black_And_Device_Tree_Overlays.php)
+3. [Sharing internet using Network-Over-USB in BeagleBone Black](https://elementztechblog.wordpress.com/2014/12/22/sharing-internet-using-network-over-usb-in-beaglebone-black/)
+
+#### Why use BeagleBone Black​ ? 
 	
 Because it's an open hardware board, it’s very convenient to use it for learning. Even electronics
```
Comment (PranabNandyNitk) [inline README.md:50]: Can we remove later part from this page

**Review by PranabNandyNitk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PranabNandy) [reply README.md:50]: We do it later.

Comment (PranabNandyNitk) [reply README.md:50]: We can put in a Booting up directory.

Comment (PranabNandy) [reply README.md:50]: Yes True.

**Review by PranabNandyNitk — APPROVED** [review]

**Pull request merged**